src/aap_eda/settings/default.py

```diff
@@ -212,11 +212,10 @@ def toggle_feature_flags(flags, settings):
 # See https://cfpb.github.io/django-flags/
 FLAGS = {}
```

Comment:
remove this line. It was due to merging conflict

Reply:
excellent catch

Reply:
Fixed.